data/src/main/java/com/teamagam/gimelgimel/data/message/repository/cache/room/mappers/FeaturesToEntityFeaturesUtilities.java

```diff
@@ -0,0 +1,48 @@
+package com.teamagam.gimelgimel.data.message.repository.cache.room.mappers;
+
+import com.teamagam.gimelgimel.data.message.repository.cache.room.entities.AlertFeatureEntity;
+import com.teamagam.gimelgimel.data.message.repository.cache.room.entities.GeoFeatureEntity;
+import com.teamagam.gimelgimel.data.message.repository.cache.room.entities.ImageFeatureEntity;
+import com.teamagam.gimelgimel.domain.alerts.entity.Alert;
+import com.teamagam.gimelgimel.domain.messages.entity.features.AlertFeature;
+import com.teamagam.gimelgimel.domain.messages.entity.features.GeoFeature;
+import com.teamagam.gimelgimel.domain.messages.entity.features.ImageFeature;
+import javax.inject.Inject;
+
+public class FeaturesToEntityFeaturesUtilities {
```

Comment:
Use the name mapper to increase readability.
 FeatureToEntityMapper for instance